GitPrDemo/Program.cs

```diff
@@ -1,3 +1,10 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+using System.Globalization;
+using GitPrDemo;
+
+var pesel = new Pesel("24252301243");
+
+CultureInfo.CurrentCulture = new CultureInfo("pl-PL");
```

Comment:
Why did you hard-code this culture? You could grab it from OS, or leave invariant.